dashboard/config/libraries/variableLog.interpreted.js

```diff
@@ -1,9 +1,25 @@
 var studentVarToken = false;
-var varLog={};
-if(!validationProps){
-  var validationProps ={};
+var varLog = buildVariableLog();
+var previousVarLog; // = JSON.parse(JSON.stringify(varLog));
```

Comment:
nit: if committing commented-out code, leave a comment explaining why, otherwise over time it gets confusing about the intent.  Ideally, though, there's no need to commit commented-out code.